### PR TITLE
🐛 Export package.json

### PR DESCRIPTION
**Pull request by matinzd**
Fixes [ERR_PACKAGE_PATH_NOT_EXPORTED] -> Add package subpath

related to https://github.com/nodejs/node/issues/33460

**_Category:_**

- [ ] ✨ Introduce new features
- [ ] 📝 Add or update documentation
- [ ] ✅ Add or update tests
- [x] 🐛 Fix a bug
- [ ] 🏷️ Add or update types
- [ ] ⚡️ Improve performance
- [ ] _Other(s):_ ...
  
  

**_Potential impacts:_**

- [ ] Generated values
- [ ] Shrink values
- [ ] Performance
- [ ] Typings
- [x] _Other(s):_ ...

### CODE REVIEW

**Comment by dubzzz** [comment]
Thanks a lot for the fix.

We should probably apply the same for 2 other packages of the monorepo that rely on exports: ava and jest. Things that you or I can do in another PR, another PR so that I can patch and backport the fix more easily for each distinct package.

For the red CI test, you will mostly need to run `yarn version:bump` and tell yarn that you expect a patch for fast-check.

**Comment by matinzd** [comment]
I just updated two other packages in the same PR. Is that ok for you? @dubzzz

File: .yarn/versions/19173632.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+releases:
+  fast-check: patch
+
+declined:
+  - "@fast-check/monorepo"
+
+undecided:
+  - "@fast-check/ava"
+  - "@fast-check/jest"
```
Comment (dubzzz) [inline .yarn/versions/19173632.yml:9]: Those two should be marked for patches too

File: .yarn/versions/19173632.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  - "@fast-check/packaged"
```
Comment (dubzzz) [inline .yarn/versions/19173632.yml:10]: This one should be marked as declined

**Review by dubzzz — COMMENTED** [review]
Minor comments

File: .yarn/versions/19173632.yml (hunk 2 above)
Comment (matinzd) [reply .yarn/versions/19173632.yml]: Done.

File: .yarn/versions/19173632.yml (hunk 1 above)
Comment (matinzd) [reply .yarn/versions/19173632.yml]: Done.

**Comment by matinzd** [comment]
@dubzzz Done.

**Pull request merged**